api/models.py

```diff
@@ -2,18 +2,16 @@
 from django.db import models
 from django.db.models import F, Q
 
+GENDER_CHOICES = (("M", "Male"), ("F", "Female"), ("O", "Other"))
```

Comment:
I'd say it is better to leave the choices as a class variable in the Profile because they are associated with the profile. [Django also recommends this](https://docs.djangoproject.com/en/3.1/ref/models/fields/#django.db.models.Field.choices). 
> Though you can define a choices list outside of a model class and then refer to it, defining the choices and names for each choice inside the model class keeps all of that information with the class that uses it, and helps reference the choices (e.g, Student.SOPHOMORE will work anywhere that the Student model has been imported)

Reply:
👍